src/Native/Unix/System.Native/pal_io.h

```diff
@@ -15,6 +15,12 @@ BEGIN_EXTERN_C
 #include <dirent.h>
 #include <sys/types.h>
 
+#if defined(_AIX) && !defined(F_DUPFD_CLOEXEC)
+// HACK: Try to get AIX 6.1 & i 7.1 working; AIX 7.1 & i (claims) 7.2 has these but we target
+// these older versions too, so nop this define out.
+#define F_DUPFD_CLOEXEC 0
```

Comment:
I am not sure how this is going to work. We use this symbol at one place only:
https://github.com/dotnet/corefx/blob/09b3cc02d85c5c963af50567cb89b6d67beebc49/src/Native/Unix/System.Native/pal_io.c#L276-L281

I am not sure how defining it as 0 would work there.

Reply:
Yeah, you're right; I wrote that when I was really ill and irritated. Let me fix that.